instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./CommentSection.css";
+
+class CommentSection extends React.Component {
+  state = {
+    comments: this.props.commentArr,
```

Comment:
Setting the comments to state in the CommentSection component does work, and it's a good approach for working with this project. But in general, you won't want to do this because it will fragment out part of your data. You will almost always need to pass the updated data back up to the original data set. You'll learn more about that with state management and redux.